lib/bundler/installer/parallel_installer.rb

```diff
@@ -35,13 +35,14 @@ def ignorable_dependency?(dep)
     # sure needed dependencies have been installed.
     def dependencies_installed?(all_specs)
       installed_specs = all_specs.select(&:installed?).map(&:name)
-      dependencies.all? {|d| installed_specs.include? d.name }
+      dependencies(all_specs.map(&:name)).all? {|d| installed_specs.include? d.name }
```

Comment:
I fixed that particular error but it looked like Travis went wonky after my last commit.